### ISSUE TITLE
array_unshift() expects parameter 1 to be array, null given

### ISSUE THREAD

**Issue by razvanphp** [erusev/parsedown-extra#37]
Hey,

Just made today composer update and we get the following error:

````
PHP Warning:  array_unshift() expects parameter 1 to be array, null given in /.../vendor/erusev/parsedown-extra/ParsedownExtra.php on line 31
PHP Fatal error:  Call to undefined method ParsedownExtra::blockDefinitionList() in /.../vendor/erusev/parsedown/Parsedown.php on line 219
````

Does this update break compatibility?
````
- Removing erusev/parsedown (1.1.4)
- Installing erusev/parsedown (1.4.2)
````

**Comment by erusev** [comment]
Can you provide the Markdown text that produced this?

**Comment by erusev** [comment]
It looks like you have updated Parsedown but not Parsedown Extra. You should upgrade both in order for ParsedownExtra to work properly.

**Comment by razvanphp** [comment]
Ok, I fixed it by updating the parsedown-extra to 0.5.1, but parsedown was only defined as a dependency in extra, so I would expect it to freeze parsedown version with a compatible one, not update freely. Thank you!
